### PR TITLE
add get user stats endpoint

### LINKED ISSUE #9: Add get user stats
[Monkeytype Issue](https://github.com/Miodec/monkeytype/issues/2892)

### PR DESCRIPTION
**Pull request by Ferotiq**
Closes #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "pretty:fix": "prettier ./ --write",
     "lint": "eslint ./src/**/*.ts",
     "lint:fix": "eslint ./src/**/*.ts --fix",
-    "test": "node test/test.js"
+    "test": "yarn build && node test/test.js"
```
Comment (Bruception) [inline package.json:15]: yarn?

File: src/endpoints/users.ts [hunk 2]
```diff
@@ -26,4 +30,12 @@ export class UsersEndpoints {
 
     return this.httpClient.get(`${BASE_PATH}/personalBests`, { query });
   }
+
+  /**
+   * Gets a your personal stats
+   * @returns Your personal stats
+   */
+  public async getStats(): Promise<ApiResponse<Stats>> {
+    return this.httpClient.get(`${BASE_PATH}/stats`, { query: {} });
```
Comment (Bruception) [inline src/endpoints/users.ts:39]: Remove: `{ query: {} }`

**Comment by Ferotiq** [comment]
@Bruception for some reason I can't re-request a review here so can you review this one again

**Review by Bruception — APPROVED** [review]
LGTM IIW

**Pull request merged**